### PR TITLE
test: metoken e2e

### PR DESCRIPTION
**Pull request by kosegor**

### CODE REVIEW

File: tests/grpc/gov.go [hunk 1]
```diff
@@ -5,6 +5,10 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/umee-network/umee/v6/x/metoken"
```
Comment (toteki) [inline tests/grpc/gov.go:8]: can group these with the other `/umee/v6` imports

File: tests/e2e/e2e_metoken_test.go [hunk 2]
```diff
@@ -0,0 +1,231 @@
+//go:build experimental
+// +build experimental
```
[the remark this answers is not in the document]
Comment (gsk967) [reply tests/e2e/e2e_metoken_test.go:2]: No, metoken is in `experimental` feature

File: tests/e2e/setup/utils.go [hunk 3]
```diff
@@ -245,6 +233,58 @@ func (s *E2ETestSuite) QueryUmeeBalance(
...
+
+func (s *E2ETestSuite) TxMetokenRedeem(umeeAddr string, meToken sdk.Coin, assetDenom string) error {
+	req := &metoken.MsgRedeem{
+		User:       umeeAddr,
+		Metoken:    meToken,
+		AssetDenom: assetDenom,
+	}
+
+	return s.broadcastTxWithRetry(req)
+}
```
[the remark this answers is not in the document]
Comment (kosegor) [reply tests/e2e/setup/utils.go:268]: but `broadcastTxWithRetry` will be used by @toteki as well as by anyone wants to broadcast tx with retry

[the remark this answers is not in the document]
Comment (kosegor) [reply tests/e2e/setup/utils.go:268]: I guess you meant queries and txs. Moving them

**Pull request merged**